### PR TITLE
AbstractVpnInterface : misc cleanup

### PR DESCRIPTION
**Pull request by andirsun**
- Object initialize refactor 
- Bump copyright header

### CODE REVIEW

File: src/common/Widgets/AbstractVpnInterface.vala [hunk 1]
```diff
@@ -49,10 +49,11 @@ public abstract class Network.AbstractVpnInterface : Network.WidgetNMInterface {
     }
 
     construct {
-        vpn_list = new Gtk.ListBox ();
         // Single click is disabled because it's being handled by VpnMenuItem
-        vpn_list.activate_on_single_click = false;
-        vpn_list.visible = true;
+        vpn_list = new Gtk.ListBox () {
+            activate_on_single_click = false;
+            visible = true;
```
Comment (ryonakano) [inline src/common/Widgets/AbstractVpnInterface.vala:55]:
```suggestion
            activate_on_single_click = false,
            visible = true
```

File: src/common/Widgets/AbstractVpnInterface.vala [hunk 2]
```diff
@@ -49,10 +49,11 @@ public abstract class Network.AbstractVpnInterface : Network.WidgetNMInterface {
...
 
     construct {
-        vpn_list = new Gtk.ListBox ();
         // Single click is disabled because it's being handled by VpnMenuItem
-        vpn_list.activate_on_single_click = false;
-        vpn_list.visible = true;
+        vpn_list = new Gtk.ListBox () {
+            activate_on_single_click = false,
+            visible = true
+        }
```
Comment (ryonakano) [inline src/common/Widgets/AbstractVpnInterface.vala]:
```suggestion
        };
```

Ah, I need to mention here too

**Comment by andirsun** [comment]
> You're faster than me

Sorry for my bads, it my first time with vala and helping here

**Comment by ryonakano** [comment]
Thar's alright, thank you for your PRs!

**Pull request merged**